src/annotations/markups/utils/getUnitsSuffix.js

```diff
@@ -0,0 +1,42 @@
+import { getPointResolution } from 'ol/proj';
+
+const UnitsEnum = { METERS: 'm' };
+const DEFAULT_DPI = 25.4 / 0.28;
+
+const getUnitsSuffix = (view) => {
```

Comment:
@hackermd I have pushed some code that keeps the ROI's measurements and evaluations generation in sync with other custom feature properties like 'label' which allows the client app to update the label of the freetext. 'area', 'length' which gets updated every change in the geometry, and 'marker' / 'markup' which defines the style and behavior of markers and markups. I think it's simpler to work with the feature properties directly which will update the corresponding ROI object. You can take a look at the AnnotationManager where all this logic lies. 

I'm not using CODE evaluations for now because our client app only required freetext working but it's not hard to do, we just need to set another property like CODE that would be stored in a separate code content item and updated like the label property.

Using OHIF, I could store and retrieve the SR as expected for measurements and freetext. Now I'm not sure how I'm going to store information related to the marker itself since this is just a presentational state. Maybe another text content item could store this info (arrow marker) that would work only for this specific application? (not ideal)

Just let me know if you prefer to schedule a meeting to show you.